docs/source/config.json

```diff
@@ -1,6 +1,10 @@
 {
   "title": "Apollo Server",
   "version": "v2",
+  "versionBanner": {
+    "link": "/apollo-server/previous-versions",
+    "content": "Apollo Server 2 is officially deprecated. **Learn more about updating to a supported Apollo Server version.**"
```

Comment:
I think we should include the EOL date in the message.

Reply:
Good call out! I'll probably use the following text:
```
"Apollo Server 2 is officially deprecated with end-of-life scheduled for 22 October 2023. **Learn more about upgrading to a supported Apollo Server version.**
```

I'm working with DXE right now just to ensure we can see what it looks like in a deploy preview before we merge it :)